azure-vote/Dockerfile

```diff
@@ -1,3 +1,13 @@
-FROM tiangolo/uwsgi-nginx-flask:python3.6
```

Comment:
please remove the commented lines and the comment. The git history is sufficient to explain the change